example/common/common.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+
+package common;
+
+import "github.com/kitt-technology/protoc-gen-graphql/graphql/graphql.proto";
```

Comment:
Circular dep here, if we separate `github.com/kitt-technology/protoc-gen-graphql/graphql/` then fixes I think

Reply:
Will also cause circular deps in the generated code if we use `google.protobuf.Timestamp` in here, think we could move to our own timestamp to avoid this